connectors proxy settings (#5010)
camunda/camunda-docs: docs/components/connectors/protocol/rest.md

| `http.nonProxyHosts` | _(optional)_ <p>A list of hosts to connect to directly, bypassing the proxy.</p><p><ul><li>Specify as a list of patterns, separated by <code>&#124</code>.</li><li>Patterns can start or end with a `*` for wildcards.</li><li>Any host matching one of these patterns uses a direct connection instead of a proxy.</li></ul></p> |
| `http.user`          | _(optional)_ The username to login to the proxy                                                                                                                                                                                                                                                                                   |
| `http.password`      | _(optional)_ The password to login to the proxy                                                                                                                                                                                                                                                                                   |

> Alternative to setting the proxy with JVM properties, the proxy settings can also be set with these environment variables:

Reviewer: ```suggestion
As an alternative to using JVM properties, the proxy settings can also be set with environment variables:
```
Rewording suggestion

| `https.password`      | _(optional)_ The password to log in to the proxy.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |

> Alternative to setting the proxy with JVM properties, the proxy settings can also be set with these environment variables:

Reviewer: ```suggestion
As an alternative to using JVM properties, the proxy settings can also be set with environment variables:
```
